### PR TITLE
fix(docker): superset permissions and firefox config

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
This PR fixes/improves the following:

- Fixes firefox config for thumbnails and alerts & report when running with a low privilege user.
- Easier way to run dev docker-compose with a low privilege user eg: superset
- Fixes superset homedir

### ADDITIONAL INFORMATION
- [X] Has associated issue: #14330
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -84,8 +84,8 @@ ENV LANG=C.UTF-8 \
     SUPERSET_HOME="/app/superset_home" \
     SUPERSET_PORT=8088
 
-RUN useradd --user-group --no-create-home --no-log-init --shell /bin/bash superset \
-        && mkdir -p ${SUPERSET_HOME} ${PYTHONPATH} \
+RUN useradd --user-group -d ${SUPERSET_HOME} --no-log-init --shell /bin/bash superset \
+        && mkdir -p ${PYTHONPATH} \
```
Comment (villebro) [inline Dockerfile:88]: I like this 👍

File: docker/docker-bootstrap.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ if [[ "${1}" == "worker" ]]; then
   celery worker --app=superset.tasks.celery_app:app -Ofair -l INFO
 elif [[ "${1}" == "beat" ]]; then
   echo "Starting Celery beat..."
-  celery beat --app=superset.tasks.celery_app:app --pidfile /tmp/celerybeat.pid -l INFO
+  celery beat --app=superset.tasks.celery_app:app --pidfile /tmp/celerybeat.pid -l INFO -s /app/superset_home/celerybeat-schedule
```
Comment (villebro) [inline docker/docker-bootstrap.sh:44]: Should we pick the path to `/app/superset_home` from `${SUPERSET_HOME}`?

Comment (dpgaspar) [reply docker/docker-bootstrap.sh]: Good point, changed it

**Review by willbarrett — APPROVED** [review]

File: superset/config.py [hunk 3]
```diff
@@ -1027,16 +1027,12 @@ def CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC(
...
 
 # Additional args to be passed as arguments to the config object
 # Note: these options are Chrome-specific. For FF, these should
 # only include the "--headless" arg
-WEBDRIVER_OPTION_ARGS = [
-    "--force-device-scale-factor=2.0",
-    "--high-dpi-support=2.0",
-    "--headless",
-]
+WEBDRIVER_OPTION_ARGS = ["--headless", "--marionette"]
```
Comment (john-bodley) [inline superset/config.py:1035]: @dpgaspar should this just be `["--headless"]` given that the `WEBDRIVER_TYPE` is defaulted to `firefox`?

**Pull request merged**